app/assets/javascripts/application.js

```diff
@@ -13,4 +13,3 @@
 //= require jquery
 //= require jquery_ujs
 //= require bootstrap-sprockets
-//= require_tree .
```

Comment:
@Bion so by removing this (^^^) line I got the app to work again, but it seems like that undermines the asset pipeline.  I made sure to get rid of all the `.coffee` extensions i could find but the app seems to be trying to call coffee script from somewhere.  The error points to line 7 of `/app/views/layouts/application.html.haml` which reads:

`= javascript_include_tag 'application'`